src/ui/components/WalletConnectRequest/WalletConnectRequest.scss

```diff
@@ -0,0 +1,194 @@
+@keyframes slidein {
```

Comment:
Please see comments above regarding renaming this component and refactoring the `Connections` component to be reusable, passing `WalletConnect` component as child.